database/factories/AddressFactory.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Faker\Generator as Faker;
+use GetCandy\Api\Core\Addresses\Models\Address;
+
+/*
+|--------------------------------------------------------------------------
+| Model Factories
+|--------------------------------------------------------------------------
+|
+| This directory should contain each of the model factory definitions for
+| your application. Factories provide a convenient way to generate new
+| model instances for testing / seeding your application's database.
+|
+*/
+
+$factory->define(Address::class, function (Faker $faker) {
+    return [
+        'salutation' => $faker->title,
+        'firstname' => $faker->firstName,
+        'lastname' => $faker->lastName,
+        'email' => $faker->safeEmail,
+        'phone' => $faker->phoneNumber,
+        'company_name' => $faker->company,
+        'address' => $faker->streetAddress,
+        'address_two' => $faker->streetName,
+        'address_three' => null,
+        'city' => $faker->city,
+        'county' => $faker->state,
+        'postal_code' => $faker->postcode,
+        'billing' => $faker->boolean,
+        'shipping' => $faker->boolean,
+        'default' => $faker->boolean,
```

Comment:
Did we want to include the ability to "favourite" some addresses, so they come first in the list?

Reply:
Do you think maybe "preferred" would be a better word?

Reply:
I think "popular" would be the best word.

Reply:
So would this be determined by how often it's used? Would this be an int that gets incremented each time then?

Reply:
I think it's just a boolean, something the store admin can set.

Reply:
I might be confused, this is for a users address not a country? why would a store admin set a "popular" address for a user?

Reply:
Oh yeah, I'm thinking about countries... haha!